docs/af5-app-from-scratch/modules/ROOT/pages/sample-domain-event-modeling.adoc

```diff
@@ -0,0 +1,39 @@
+:navtitle: Sample Domain & Event Modeling
+:reftext: Sample Domain & Event Modeling
+
+== The sample domain
```

Comment:
There is no top-level header here, so the name of the page is not rendered in the nav